pkg/proxy/ipvs/proxier.go

```diff
@@ -2008,6 +2008,9 @@ func (proxier *Proxier) syncService(svcName string, vs *utilipvs.VirtualServer,
 func (proxier *Proxier) syncEndpoint(svcPortName proxy.ServicePortName, onlyNodeLocalEndpoints bool, vs *utilipvs.VirtualServer) error {
 	appliedVirtualServer, err := proxier.ipvs.GetVirtualServer(vs)
 	if err != nil || appliedVirtualServer == nil {
+		if err == nil {
+			err = fmt.Errorf("non-existent IPVS service")
```

Comment:
Prefer just to break the above if into two blocks:
```
if err != nil {
    return fmt.Errorf("error getting IPVS virtual service: %w", err)
}

if appliedVirtualServer == nil {
   return errors.New("IPVS virtual service does not exist")
}
```

Reply:
Thanks for review! code has already modified according to this comment